ctwrap/strategy.py

```diff
@@ -79,6 +81,52 @@ def _sweep_matrix(tasks, defaults):
     return out
 
 
+def _parse_mode(strat_dict):
+    """
+    parse the strategy/variation values based on the mode specified
+
+    Argument:
+        strat_dict(dict): variation values specification
+
+    Output:
+        out (dict): parsed variation values
+
+    """
+    k, strat_val = list(strat_dict.items())[0]
+    strat_limits = strat_val['limits']
+    strat_max = strat_limits[1]
+    strat_min = strat_limits[0]
+    strat_rev = strat_val.get('reverse', False)
+    if strat_min and strat_max and strat_min > strat_max:
+        msg = ("Minimum strategy value is larger than maximum: use "
+               "keyword 'reverse' instead")
+        warnings.warn(msg, PendingDeprecationWarning)
+        strat_min, strat_max = strat_max, strat_min
+        strat_rev = True
+    if "mode" not in strat_val:
```

Comment:
separate different `if` conditional block by an empty line. It becomes easier to follow.